src/logger/console.logger.ts

```diff
@@ -0,0 +1,15 @@
+import { Logger } from "./logger.types"
+
+// eslint-disable-next-line import/prefer-default-export
+export const consoleLogger: Logger = {
+  info: (message: string | Record<string, unknown>): void => {
+    // NOTE: This adds RGB to our console logs
+    console.log("\x1b[32m%s\x1b[0m", `[INFO]: ${message}`)
```

Comment:
[Nit, style]: could we name the colours? 
eg.
```
const redColour = \x1b[32m%s\x1b[0m
console.log(redColour, `[INFO]: ${message}`)
```

Reply:
sounds good!